backend/src/contract/kanban/KanbanStoryPointResponse.ts

```diff
@@ -1,6 +1,7 @@
-import { JiraCard } from "../../models/kanban/JiraCard";
+import { JiraCard, Status } from "../../models/kanban/JiraCard";
```

Comment:
could you please remove the unused import?
same issue in Issue

Reply:
done.